scan-triggers/aws-python-bucket-full-and-scheduled-scan/template.yaml

```diff
@@ -856,7 +862,11 @@ Resources:
               \      }
 
               \      "
-      StateMachineName: fullScanStarterStateMachine
+      StateMachineName:
+        Fn::Join:
+          - ""
+          - - fullScanSSM-
```

Comment:
Why is it `SSM` but not `SM`? How about we just remove the acronym, since we already know that's a state machine/step function? By the way, would you please also use capital for the first char, like `ScannerLoop`?